internal/analysis/rundynamic.go

```diff
@@ -0,0 +1,93 @@
+package analysis
+
+import (
+	"github.com/ossf/package-analysis/internal/dynamicanalysis"
+	"github.com/ossf/package-analysis/internal/log"
+	"github.com/ossf/package-analysis/internal/pkgecosystem"
+	"github.com/ossf/package-analysis/internal/sandbox"
+)
+
+type DynamicAnalysisResults map[pkgecosystem.RunPhase]*dynamicanalysis.Result
+
+// RunDynamicAnalysis runs dynamic analysis on the given package in the sandbox
+// provided, across all phases (e.g. import, install) valid in the package ecosystem.
+// Status and errors are logged to stdout. There are 3 return values:
+// results: Map of each successfully run phase to the corresponding dynamic analysis result.
+// If error is not nil, then results[lastRunPhase] is nil.
+// lastRunPhase: the last phase that was run. If error is non-nil, this phase did not complete successfully
+// and the results for this phase are not recorded. Otherwise, results[lastRunPhase] contains
+// the corresponding results this phase, including any abnormal termination of the sandboxed process.
+// error: Any error that occurred in the runtime/sandbox infrastructure. This does not include errors caused
+// by the package under analysis.
```

Comment:
Nit: some blank/whitespace lines may help make this comment easier to read.

Reply:
Sure